db/migrate/20200318135632_rename_report_templates.rb

```diff
@@ -0,0 +1,30 @@
+class RenameReportTemplates < ActiveRecord::Migration[5.2]
+  TEMPLATE_NAMES = {
+    "Ansible Inventory" => "Ansible - Ansible Inventory",
+    "Applicable errata" => "Host - Applicable Errata",
+    "Applied Errata" => "Host - Applied Errata",
+    "Entitlements" => "Subscription - Entitlement Report",
+    "Host statuses" => "Host - Statuses",
+    "Registered hosts" => "Host - Registered Content Hosts",
+    "Registered users" => "User - Registered Users",
+    "Subscriptions" => "Subscription - General Report",
+  }
+
+  class FakeReportTemplate < ApplicationRecord
+    self.table_name = 'templates'
+  end
+
+  def up
+    TEMPLATE_NAMES.each do |from, to|
+      FakeReportTemplate.find_by_name(to).delete
+      FakeReportTemplate.find_by_name(from).try(:update_attributes, :name => to)
```

Comment:
Does this mean that if the user has for example "Ansible - Ansible Inventory" as a custom report template, the migration will just overwritte it with what was previously "Ansible Inventory" known as?

Reply:
That is indeed what would happen, do we have some way out of it @lzap ?

Reply:
Well I guess I can create a backup with some random characters in the name so it's high probability it's unique if you insist. But since we always recommend a backup for major updates, is this really a problem?

Reply:
Well, I think it's user unfriendly to delete user created template, they can notice like week after using new version and then it's not as simple to get it back.
I :heart: your solution, easy and simple and no harm done to any user! :+1: